restdocs-api-spec-jsonschema/src/main/kotlin/com/epages/restdocs/apispec/jsonschema/JsonSchemaFromFieldDescriptorsGenerator.kt

```diff
@@ -69,10 +69,13 @@ class JsonSchemaFromFieldDescriptorsGenerator {
         val objectMapper = jacksonObjectMapper().enable(SerializationFeature.INDENT_OUTPUT)
         return StringWriter().use {
             schema.describeTo(JSONPrinter(it))
-            objectMapper.writeValueAsString(objectMapper.readTree(it.toString()))
+            val rawJsonSchema = cleanupRawJsonSchema(it.toString())
+            objectMapper.writeValueAsString(objectMapper.readTree(rawJsonSchema))
         }
     }
 
+    private fun cleanupRawJsonSchema(rawJsonSchema: String) = rawJsonSchema.replace("\"enum\"", "\"type\":\"string\",\"enum\"")
```

Comment:
Nice hack! ;) See the risk above. So 
```
"enum": ["FOO", "BAR"]
```
would become
```
"type": "string","enum": ["FOO", "BAR"]
```

Imo, it would be better to check on the object structure whether it has an "enum" field, if so, insert the field `"type"` with value `"string"`